src/pages/[userId]/prospects.js

```diff
@@ -1,54 +1,148 @@
 import { useRouter } from 'next/router';
-import Grid from '../../components/utils/Grid';
```

Comment:
You can also delete our old custom `Grid` component along with the `utils` directory.

Reply:
I'll need to refactor the navbar first since it uses Grid. I'll remove it in the minister branch since it also needs to be refactored.

Reply:
Oh good point haha 👍